runtime/doc/lua.txt

```diff
@@ -849,6 +849,61 @@ vim.types						*vim.types*
 	`vim.types.dictionary` will not change or that `vim.types` table will 
 	only contain values for these three types.
 
+==============================================================================
+Lua module: vim dicts                                           *lua-vim-dicts*
```

Comment:
I think that "lua-vim-internal-variables" is better. (And maybe add a section for options)
"internal-variables" is Vim variables help index name.
(And also, In the case of Lua, it should be called a table instead of a dictionary.)